fdbclient/SystemData.cpp

```diff
@@ -275,6 +275,7 @@ std::pair<vector<std::pair<UID, NetworkAddress>>, vector<std::pair<UID, NetworkA
 
 const KeyRef serverKeysPrefix = LiteralStringRef("\xff/serverKeys/");
 const ValueRef serverKeysTrue = LiteralStringRef("1"), // compatible with what was serverKeysTrue
+    serverKeysTrueEmptyRange = LiteralStringRef("3"), // the server treats the range as empty.
```

Comment:
The newer code would prefer `"3"_sr` over `LiteralStringRef("3")`.

Reply:
Done, thanks for the tip. Also changed the existing key/value in the same group.